src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -264,6 +270,9 @@ const ActionsPage = () => {
 
   return (
     <div className={styles.main}>
+      {actionType === ColonyActions.Recovery && (
+        <p className={styles.recoveryTag}>{formatMessage(MSG.recoveryTag)}</p>
```

Comment:
Please use the  `FormattedMessage` component where possible, rather than the programatic `formatMessage`.

```suggestion
        <p className={styles.recoveryTag}>
          <FormattedMessage {...MSG.recoveryTag} />
        </p>
```

Reply:
The programmatic hook is a legacy leftover in `react-intl` and is mostly used for stuff like giving a component the ability to support both strings and message descriptors as values.

See this as an example:
https://github.com/JoinColony/colonyDapp/blob/6a8ca7f612960c8c7788833202eceb3f5a709216/src/modules/core/components/Heading/Heading.tsx#L55-L65